packages/@lwc/ssr-compiler/package.json

```
@@ -51,9 +51,11 @@
         "astring": "^1.9.0",
         "estree-toolkit": "^1.7.8",
         "immer": "^10.1.1",
-        "meriyah": "^5.0.0"
+        "meriyah": "^5.0.0",
+        "@babel/types": "7.26.0"
     },
     "devDependencies": {
-        "@types/estree": "^1.0.6"
+        "@types/estree": "^1.0.6",
+        "@types/babel-types": "^7.0.16"
```

```suggestion
        "@types/estree": "^1.0.6",
```
We shouldn't need `@types/babel-types`, because that's for `babel-types` and we're using `@babel/types`. (Unless there's something weird going on with nested dependencies?)

---

Good call, thanks